### ISSUE TITLE
[Accessibility issue] Vignettes lacks alt= in plots

### ISSUE THREAD

**Issue by MichaelChirico** [easystats/easystats#352]
We have a tool that flags images that lack an `alt=` attribute. See https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices -- these are essential for conveying the image contents to users on screen readers.

You can add these to your vignettes by using the recent-ish `fig.alt` chunk option in `knitr`; see https://www.rstudio.com/blog/knitr-fig-alt/.

The vignettes at issue are https://github.com/easystats/insight/blob/main/vignettes/insight.Rmd (5x) and https://github.com/easystats/insight/blob/main/vignettes/export.Rmd (1x)

Unfortunately the lines are flagged in the HTML output, which won't correspond exactly to the .Rmd file lines, but I assume if you're familiar with the vignette it will be clear which images need it.

Here's a dump of the output on the HTML vignettes:

```
insight/inst/doc/export.html:426
# ---
insight/inst/doc/insight.html:387
insight/inst/doc/insight.html:420
insight/inst/doc/insight.html:429
insight/inst/doc/insight.html:438
insight/inst/doc/insight.html:448
```

**Comment by IndrajeetPatil** [comment]
Also related to https://github.com/easystats/see/issues/127.
